packages/framer-motion/src/render/types.ts

```diff
@@ -35,6 +35,7 @@ export interface VisualElement<Instance = any, RenderState = any>
     measureViewportBox(withTransform?: boolean): Box
     addVariantChild(child: VisualElement): undefined | (() => void)
     getClosestVariantNode(): VisualElement | undefined
+    shouldReduceMotion?: boolean | null
```

Comment:
What is the difference between `undefined` and `null` here?

Reply:
`null` is provided server-side, basically in situations where `window` doesn't exist. The API here is pre-existing in `useReducedMotion`